src/components/reusable/tabs/tabs.ts

```diff
@@ -29,6 +30,16 @@ export class Tabs extends LitElement {
   @property({ type: Boolean })
   vertical = false;
 
+  /** Enables tab content change on focus with keyboard navigation/assistive technologies. . */
+  @property({ type: Boolean })
+  autoFocusUpdate = false;
+
+  /** Changes tab content.
+   * @internal
+   */
+  @state()
+  changeTabContent = false;
```

Comment:
I don't think we need two different variables to control this. Can it be handled with just `autoFocusUpdate`?

Reply:
Done